### ISSUE TITLE
Feature: Pass tags to s3 buckets in s3-archive module

### ISSUE THREAD

**Issue by HansBraun** [coralogix/terraform-coralogix-aws#88]
Since we control a lot of our infrastructure via tags, it is very important for us to be able to set them explicitly. Unfortunately, in the current version of the s3-archive module it is not possible to pass tags, which are then set for the metrics and log buckets.

**Comment by MichaelBriggs-Coralogix** [comment]
@HansBraun 
Is it important for you to tag the two s3 buckets differently? We have several of our TF modules with support for "global" tags, where tags will be applied to all created resources, identically. We easily add that same support to this module. But, it sounds like you need individual tags though, per resource. Is that correct?

**Comment by MichaelBriggs-Coralogix** [comment]
@HansBraun 
Thanks for your update. While investigating this issue, I found that the AWS Provider module already provides support for tagging of all elements with a default tag as detailed here:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/resource-tagging#propagating-tags-to-all-resources
It looks like adding support for this use case is no longer necessary, as it's available at the provider level.

**Comment by MichaelBriggs-Coralogix** [comment]
@HansBraun

Does the AWS Provider tagging route work for you? Were you able to get that working for your use case?

**Comment by HansBraun** [comment]
@MichaelBriggs-Coralogix  Yes, it's working in my use case, as I am fine with tagging both buckets equally.
